doc/api.md

```diff
@@ -60,89 +45,68 @@ Reference table for genes within Cognoma. Entire model is read-only.
 | standard_name | string |  |
 | description | string |  |
 | organism | integer | Foreign Key referencing the Organism. |
-| aliases | string |  |
+| aliases | string | space-separated list of aliases |
 | obsolete | boolean |  |
-| weight | boolean |  |
-| sample_mutation_sumary | object | Count/aggregates of sample mutations available in Cognoma for this particular gene. |
-
-#### Tissues (/tissues)
-
-Entire model is read-only.
-
-| Field        | Type           | Description |
-| ------------- |:-------------:| ----------:|
-| acronym | string | Primary Key. Ex "GBM" |
-| name | string | Full tissue name. Ex "Glioblastoma Multiforme" |
+| weight | boolean | Weight used for search results since multiple genes may have the same symbol |
 
 ### Organism (/organisms)
 
-Reference table for organisms within Cognoma. Entire model is read-only.
-
+Reference table for organisms within Cognoma. Entire model is read-only. This table is created using [`django-organisms`](https://bitbucket.org/greenelab/django-organisms). Note that Project Cognoma will deal exclusively with human genes (when taxonomy_id equals 9606).
 
 | Field        | Type           | Description |
 | ------------- |:-------------:| ----------:|
 | taxonomy_id | integer | Primary Key. Taxonomy ID assigned by NCBI. |
 | common_name | string | Organism common name, e.g. 'Human' |
 | scientific_name | string | Organism scientific/binomial name, e.g. 'Homo sapiens' |
 
-### Sample Mutation Summary (embedded in Gene)
+### Disease Types (/diseases)
```

Comment:
With this setup "disease" is in the long format (e.g. Glioblastoma Multiforme) - alternatively, we could also use "acronym" (e.g. GBM).